src/Pages/NotesPage/NotesPage.css

```diff
@@ -1,4 +1,12 @@
 
+.main-container-wrapper{
+    /* background-color: cadetblue; */
+    width: '70%',;
```

Comment:
Suggestion: remove this apostrophe from width.

Reply:
removed